frontend-ssr/components/feedback-review/comment-block.module.scss

```diff
@@ -0,0 +1,48 @@
+.comment {
+  background-color: #fff;
+  border: 1px solid #efefef;
```

Comment:
Are colors among SCSS variables? If not, maybe add them there.

Reply:
I used the colors from the variables file. I've also added one myself - `light-grey`